.rustfmt.toml

```diff
@@ -0,0 +1,24 @@
+# stable options
+use_small_heuristics = "Default"
+force_explicit_abi = true
+hard_tabs = false
+max_width = 100
+merge_derives = false
+newline_style = "Auto"
+remove_nested_parens = true
+reorder_imports = true
+reorder_modules = true
+tab_spaces = 4
+use_field_init_shorthand = true
+use_try_shorthand = true
+edition = "2018"
+
+# unstable options (for one-off fixes and CI)
+merge_imports = true
+wrap_comments = true
+comment_width = 80
+condense_wildcard_suffixes = true
+format_strings = true
```

Comment:
The default is `false` but I changed this to `true`, thinking that it would lead to more compact, easier to read code.  I don't have strong feelings on this one way or the other though.
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#format_strings

Reply:
I like having everything wrapped at the same column, so I'm in favor of this.